test/jdk/java/io/File/GetXSpace.java

```diff
@@ -397,4 +409,44 @@ public static void main(String[] args) throws Exception {
     // size[3]  usable space: number of bytes available to the caller
     //
     private static native boolean getSpace0(String root, long[] space);
+
+    private static native boolean isCDDrive(String root);
+
+    private static long[] getCDDriveSpace(String root) throws IOException {
```

Comment:
Why not pass the `size` array in to `getCDDriveSpace` as it is already allocated, instead of allocating again and returning?

Reply:
That makes sense. I fixed it. Thanks.

Reply:
Thanks.